### PR TITLE
chore: add link to example configmaps for pipelines to README of each

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
chore: add link to example configmaps for pipelines to README of each pipeline

this commit adds a link to example configmaps for each pipeline in each pipeline's README.

**Release note**:
```
NONE
```
@akrejcir, @jcanocan, @0xFelix

### CODE REVIEW

**Comment by jcanocan** [comment]
Nice addition! It looks good to me, I'm not dropping the lgtm tag jet to give the chance to others to take a look.

**Comment by ksimon1** [comment]
> It's just a nit but how about making it a second sentence following the first sentence of the paragraph instead?

Updated

File: release/pipelines/windows-bios-installer/README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ The Pipeline implements this by spinning up a new VirtualMachine which boots fro
 ## How to run
 
 Before you create PipelineRuns, you must create ConfigMaps with an autounattend.xml in the same namespace in which the VirtualMachine will be created.
+Examples of ConfigMaps can be found [here](https://github.com/kubevirt/kubevirt-tekton-tasks/tree/main/release/pipelines/windows-bios-installer/configmaps)
```
Comment (0xFelix) [inline release/pipelines/windows-bios-installer/README.md:44]: super super nit: Add a dot at end of sentence?

Comment (ksimon1) [reply release/pipelines/windows-bios-installer/README.md]: fixed :D

**Pull request merged**